store/src/java/com/zimbra/cs/mailbox/MailboxManager.java

```diff
@@ -831,7 +761,7 @@ private Mailbox createMailbox(OperationContext octxt, Account account, boolean s
             boolean isGalSyncAccount = AccountUtil.isGalSyncAccount(account);
             synchronized (this) {
                 // check to make sure the mailbox doesn't already exist
-                mailboxKey = mailboxIds.get(account.getId().toLowerCase());
+                mailboxKey = cacheManager.getMailboxKey(account.getId().toLowerCase());
```

Comment:
Mixing calls to `String#toLowerCase()` here. See https://github.com/Zimbra/zm-mailbox/pull/489/files#diff-56c76244b9b71b66c964d2a7e3e4844dR711 where you do not call `String#toLowerCase()`. If `String#toLowerCase()` needs to be called it should be done in the implementation of `CacheManager#getMailboxKey`.